### DISCUSSION TITLE
Is there a way to temporarily (for current search) enable flash?

### DISCUSSION THREAD

**Discussion by MunifTanjim** [folke/flash.nvim#168] [Q&A] [answered]
I want to do `/` search normally without `flash.nvim`. So I have this:

```lua
opts = {
  modes = {
    search = {
      enabled = false,
    },
  },
}
```

But every once in a while, I want to enable `flash.nvim` when searching. I can do that using:

```lua
require("flash").toggle()
```

But that enables `flash.nvim` permanently, so the next time I do `/` search, `flash.nvim` will remain enabled.

Is there a way to enable `flash.nvim` temporarily, only for the current search?

**Comment by MunifTanjim** [comment, marked as answer]
I'm doing this currently.

```lua
    local augroup = vim.api.nvim_create_augroup("plugins.flash", { clear = true })
    vim.api.nvim_create_autocmd("CmdlineLeave", {
      group = augroup,
      pattern = "/",
      callback = function()
        require("flash").toggle(false)
      end,
    })
```

Wondering if there's any built-in support for this.

**Reply by folke** [reply]
That should do the trick. not builtin way. It actually used to behave as you want, but I changed it to its current behavior since that's what some people seemed to prefer.
